controller/src/main/java/io/pravega/controller/store/PravegaTablesStoreHelper.java

```diff
@@ -357,7 +357,7 @@ public void invalidateCache(String table, String key) {
      */
     public <T> CompletableFuture<VersionedMetadata<T>> getEntry(String tableName, String key, Function<byte[], T> fromBytes,
                                                                 long requestId) {
-        log.trace(requestId, "get entry called for : {} key : {}", tableName, key);
+        log.info(requestId, "get entry called for : {} key : {}", tableName, key);
```

Comment:
Is INFO appropriate? Will this pollute logs? Maybe DEBUG is better if this is invoked frequently.